pkg/prometheus/operator.go

```diff
@@ -1165,64 +1187,88 @@ func (c *Operator) sync(ctx context.Context, key string) error {
 	}
 
 	ssetClient := c.kclient.AppsV1().StatefulSets(p.Namespace)
+
 	// Ensure we have a StatefulSet running Prometheus deployed.
```

Comment:
Lets adjust this to say:
```suggestion
	// Ensure we have a StatefulSet running Prometheus deployed and that StatefulSet names are created correctly.
```

Reply:
Done